mock_log_client.go

```diff
@@ -4,9 +4,9 @@
 package trillian
 
 import (
-	"github.com/golang/mock/gomock"
-	"golang.org/x/net/context"
-	"google.golang.org/grpc"
+	gomock "github.com/golang/mock/gomock"
+	context "golang.org/x/net/context"
+	grpc "google.golang.org/grpc"
```

Comment:
These mocks are auto generated by `mockgen` (I don't know why it decided to do this, other than perhaps being a newer version of the tool?)